src/org/benf/cfr/reader/entities/attributes/AttributeMethodParameters.java

```diff
@@ -0,0 +1,55 @@
+package org.benf.cfr.reader.entities.attributes;
+
+import org.benf.cfr.reader.entities.constantpool.ConstantPool;
+import org.benf.cfr.reader.util.bytestream.ByteData;
+import org.benf.cfr.reader.util.collections.ListFactory;
+import org.benf.cfr.reader.util.output.Dumper;
+
+import java.util.List;
+
+public class AttributeMethodParameters extends Attribute {
+    public static final String ATTRIBUTE_NAME = "MethodParameters";
+
+    private static final long OFFSET_OF_ATTRIBUTE_LENGTH = 2;
+    private static final long OFFSET_OF_REMAINDER = 6;
+    private static final long OFFSET_OF_PARAMETERS_COUNT = 6;
+    private static final long OFFSET_OF_PARAMETERS = 7;
+
+    private final int length;
+    private final int count;
+    private final List<MethodParameterEntry> parameters = ListFactory.newList();
+
+    public AttributeMethodParameters(ByteData raw, ConstantPool cp) {
+        this.length = raw.getS4At(OFFSET_OF_ATTRIBUTE_LENGTH);
+        this.count = raw.getU1At(OFFSET_OF_PARAMETERS_COUNT);
+        long offset = OFFSET_OF_PARAMETERS;
+        for (long i = 0; i < count; i++) {
+            int nameIndex = raw.getU2At(offset);
+            int accessFlags = raw.getU2At(offset + 2);
+            parameters.add(new MethodParameterEntry(cp.getEntry(nameIndex), accessFlags));
```

Comment:
`nameIndex` may be 0, indicating `null`. `getEntry` does not accept 0 arguments:
https://github.com/FabricMC/cfr/blob/dd14c75325e7c5a442f846036dc6d77682aba30b/src/org/benf/cfr/reader/entities/constantpool/ConstantPool.java#L155